packages/utils/src/Uint32SparseSet/Uint32SparseSet.ts

```diff
@@ -87,7 +87,12 @@ export function sparseSetRemove(sparseSet: IUint32SparseSet, value: number): voi
 }
 
 export function sparseSetGrow(sparseSet: IUint32SparseSet, newCapacity: number): void {
-	growBuffer(sparseSet[$buffer], newCapacity);
+	if (isGrowResizeSupported()) {
+		growBuffer(sparseSet[$buffer], newCapacity);
+	} else {
+		sparseSet[$buffer] = growBufferPolyfill(sparseSet[$buffer], newCapacity);
+	}
+
```

Comment:
I think polyfill should be moved into `growBuffer` so that it can be safely reused wherever we need to grow a buffer in the code base. Instead of a polyfill we should call it what it is, a different grow strategy. I don't know what you would call it, grow and replace? I'm sure there is a technical term I don't know. For testing, there should be an optional param for `growBuffer` that if true always grows and replaces, then we can test both in utils.